packages/libz/libz.spec

```diff
@@ -0,0 +1,60 @@
+Name: %{_cross_os}libz
+Version: 1.2.11
+Release: 1%{?dist}
+Summary: Compression and decompression library
+URL: https://www.zlib.net/
+License: zlib and Boost
```

Comment:
This needs to be the SPDX license identifier, so `Zlib` in this case:
https://spdx.org/licenses/Zlib.html

I'm not sure we ship anything that requires us to include the Boost license.